src/Components/Node/Node.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { NodeType } from "../../types";
+
+type Props = {
+  node: NodeType;
+};
+
+const TAG_MAP = {
+  header1: "h1",
+  header2: "h2",
+  header3: "h3",
+  paragraph: "p",
+  hyperlink: "a",
+  numbered_list: "ol",
+  bullet_list: "ul",
+  list_item: "li",
+  text: React.Fragment,
+  bold: "strong",
+  italic: "em",
+  underline: "u",
+  break: "br",
+  image: "img",
+  video: "video",
+};
+
+function Node({ node }: Props) {
+  const { attributes, nodes, type: nodeType, text } = node;
+  const NodeTag = TAG_MAP[nodeType];
+
+  // Handle self-closing tags
+  if (NodeTag === "img" || NodeTag === "br") {
+    return <NodeTag {...attributes} />;
+  }
```

Comment:
I am sure there is a better way to handle if the element is a self-closing tag. I tried different things and came up with conditionally rendering the self-closing tag.